src/FirmwarePlugin/FirmwarePlugin.h

```diff
@@ -291,6 +291,9 @@ class FirmwarePlugin : public QObject
     /// @return true: vehicle has gimbal, false: gimbal support unknown
     virtual bool hasGimbal(Vehicle* vehicle, bool& rollSupported, bool& pitchSupported, bool& yawSupported);
 
+    /// Used to check if running firmware is latest stable version.
+    virtual void checkIfIsLatestStable(Vehicle* vehicle);
```

Comment:
Use Doxygen or anything that is compatible with the documentation style used here.